sdk/go/src/sawtooth_sdk/messaging/stream.go

```diff
@@ -20,11 +43,23 @@ type sendRequest struct {
 	IsResponse bool
 }
 
+// Response is used to wrap a validator message coming of a channel returned by
```

Comment:
Spelling?
" Response is used to wrap a validator message coming **of** a channel returned by" 

Did you mean "off" a channel, or "from a channel"?